docs/graphql/manual/schema/enums.rst

```diff
@@ -121,11 +133,25 @@ not an already-tracked table should be used as an enum:
     }
   }
 
+Using an enum table
+^^^^^^^^^^^^^^^^^^^
+
+To set a field of a table as an enum, we need to set a reference from it to the enum table via a foreign key.
+
+**For example**, to update our ``users`` table to reference the ``user_role`` enum table:
+
+.. code-block:: sql
+
+  ALTER TABLE users ADD CONSTRAINT
+    users_role_fkey FOREIGN KEY (role) REFERENCES user_role;
+
 Make queries using enum values
```

Comment:
Can we say "making" here to make it compatible with the previous headings? I know this is not your change but it just came to my attention :)

Reply:
Fair point. Made the update